src/guide/contributing-to-an-open-jdk-project.md

```diff
@@ -96,6 +98,10 @@ Java and the JDK are very popular products, and just about every Java developer
 
 For reasons like these it’s quite possible that your change, even though it adds value to you, isn’t deemed to add enough value to the larger community.
 
+If you're relatively new to the Java platform then we recommend that you gain more experience writing Java applications before you attempt to work on the JDK itself. The purpose of the sponsored-contribution process is to bring developers who already have the skills required to work on the JDK into the existing development community. The members of that community have neither the time nor the patience required to teach basic Java programming skills or platform implementation techniques.
```

Comment:
"members of that community" -> "members of (the|this) community"

Reply:
Fixed.